justfile

```diff
@@ -42,7 +42,7 @@ action-tests test_name='Test_ProgramAction':
   export KONA_CLIENT_PATH="{{justfile_directory()}}/target/release-client-lto/kona"
 
   cd monorepo/op-e2e/actions/proofs && \
-    go test -run "{{test_name}}" -v ./...
+    go test -run "{{test_name}}" -v -count=1 ./...
```

Comment:
Why do we set `count` to 1 here?

Reply:
To make sure the tests always run with no cached results.